std/experimental/allocator/building_blocks/bitmapped_block.d

```diff
@@ -142,17 +143,49 @@ struct BitmappedBlock(size_t theBlockSize, uint theAlignment = platformAlignment
     in bytes.
 
     $(UL
-    $(LI If $(D ParentAllocator) is $(D NullAllocator), only the constructor
-    taking $(D data) is defined and the user is responsible for freeing $(D
-    data) if desired.)
-    $(LI Otherwise, both constructors are defined. The $(D data)-based
-    constructor assumes memory has been allocated with the parent allocator.
-    The $(D capacity)-based constructor uses $(D ParentAllocator) to allocate
+    $(LI If `ParentAllocator` is `NullAllocator`, only the constructor taking
+    `data` is defined and the user is responsible for freeing `data` if desired.)
+    $(LI Otherwise, both constructors are defined. The `data`-based constructor
+    assumes memory has been allocated with the parent allocator. The underlying
+    `data` must be of $(LREF OpaquePointer) type for a safe allocation of objects
+    that may contain pointers, and thus for a safe allocator.
+    The `capacity`-based constructor uses `ParentAllocator` to allocate
     an appropriate contiguous hunk of memory. Regardless of the constructor
-    used, the destructor releases the memory by using $(D
-    ParentAllocator.deallocate).)
+    used, the destructor releases the memory by using `ParentAllocator.deallocate`.)
     )
     */
+    this(OpaquePointer[] data)
+    {
+        immutable a = data.ptr.effectiveAlignment;
+        assert(a >= size_t.alignof || !data.ptr,
```

Comment:
No need for this assert, so you can delete it and also the line that defines `a`.